### DISCUSSION TITLE
Are products and collections removed from Statamic after they are removed from shopify?

### DISCUSSION THREAD

**Discussion by sunscreem** [statamic-rad-pack/shopify#250] [Help]
### Description

Title says it all really - I think it's a "one-way" import by look at the source-code?

### Environment

n/a

**Comment by ryanmitchell** [comment]
Yes they are - see : https://github.com/statamic-rad-pack/shopify/blob/main/src/Http/Controllers/Webhooks/CollectionDeleteController.php and https://github.com/statamic-rad-pack/shopify/blob/main/src/Http/Controllers/Webhooks/ProductDeleteController.php
